src/pages/nickname.html

```diff
@@ -12,7 +12,15 @@
     <main class="container">
         <h3 class="container__title">Enter your nickname:</h3>
         <input class="nickname-input" type="text" name="nickname" value="" placeholder="nickname">
-        <button class="btn">Enter!</button>
+        <a href="question.html" style="text-decoration: none;color:inherit">
+        <button class="btn">Enter!</button></a>
     </main>
+    <script defer type="module">
+        import {saveToLocalStorage, loadFromLocalStorage} from "../js/localstorage.js"
+
+        const button = document.querySelector("button");
+        button.addEventListener("click", function () {
+        saveToLocalStorage(document.querySelector('input').value, "enterName");});
+    </script>
```

Comment:
As I told before, HTML files are to hold HTML, not JS. JS files are to hold JS.
Do this like you do it in `questionPage.js`

Reply:
Done.